### PR TITLE
Removed unused LAND type from terrains

### PR DESCRIPTION
**Pull request by krs0**

### CODE REVIEW

**Comment by DjWarmonger** [comment]
`LAND` property is not useless. VCMI also allows custom `WATER` terrains, which are not `LAND`.

**Comment by krs0** [comment]
This you need to see with @IvanSavenko :)

File: hota/mod.json [hunk 1]
```diff
@@ -6,10 +6,11 @@
 	"contact" : "http://forum.vcmi.eu/viewtopic.php?t=748",
 	"licenseName" : "Creative Commons Attribution-ShareAlike",
 	"licenseURL" : "http://creativecommons.org/licenses/by-sa/4.0/deed",
-	"version" : "1.6.12",
+	"version" : "1.6.13",
 	"downloadSize" : 114.9,
 	"changelog" : 
 	{
+		"1.6.13"   : [ "Removed unavailable LAND type from terrains." ],
```
Comment (Nordsoft91) [inline hota/mod.json:13]: I think this PR gives zero improvement (and changes) to the players, so why we should show them update available and ask to download 100+Mb?

Comment (krs0) [reply hota/mod.json:13]: Improvement is that there is no strange warning in client console.
Proposal on how to handle this?

Comment (Nordsoft91) [reply hota/mod.json:13]: But why do we have warning? Could you please paste the warning?

Comment (krs0) [reply hota/mod.json:13]: Was something like element type 0 not supported blah blah....

image

Comment (Nordsoft91) [reply hota/mod.json:13]: Understood, thanks. So, it was renamed. If there are more changes planned for hota mod prior 1.4 (probably hota quest gates?) release, I suggest do not bump version.

Comment (DjWarmonger) [reply hota/mod.json:13]: Okay, I see. We assume that the terrain is land if it's not water.

```
bool TerrainType::isLand() const
{
	return !isWater();
}

```
I can hardly imagine both LAND and WATER flags at same time, but who knows :P

Comment (krs0) [reply hota/mod.json:13]: What about `clouds`? with `sky ships`?

**Pull request merged**